### ISSUE TITLE
Use ALT key for Symbols

### ISSUE THREAD

**Issue by JSylvia007** [ZitaoTech/BB9900-USB_BLE_Keyboard#2]
Might it be possible to use the ALT key for the symbols instead of having to toggle the mode?  I was a longtime blackberry user and now that the keyboard is in my hands, it feel very familiar and I want to use it that way.

Thanks for an AMAZING product!!

**Comment by ZitaoTech** [comment]
Yes, there is a work from someone else trying to type on the keyboard as close as to the original BB style.

You can browse that in this link: https://github.com/Drexel-Macintosh/BlueBerry_Q10. 

You can copy his keymap and paste into the 9900 config file like the tutorial in this page: https://github.com/ZitaoTech/zmk-config_9900

If there is something that you don't know please contact me.

**Comment by JSylvia007** [comment]
> Yes, there is a work from someone else trying to type on the keyboard as close as to the original BB style.
> 
> You can browse that in this link: https://github.com/Drexel-Macintosh/BlueBerry_Q10.
> 
> You can copy his keymap and paste into the 9900 config file like the tutorial in this page: https://github.com/ZitaoTech/zmk-config_9900
> 
> If there is something that you don't know please contact me.

This worked perfectly!!
